components/provisioner/e2e_test/test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
Calling this script should be now the only step needed for running the test locally as well, right?
If so, could this be added to the README?